src/QsCompiler/Tests.Compiler/TestCases/LinkingTests/Generics.qs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit: Do you mind running the replacement "\t" to "    " (Ctrl + h and then click on the little star to enable regex)? Thanks!

Reply:
Will do! I also have noticed that in some fs files there are lots of spaces at the ends of lines. I have been removing those with: <space>+(?=[\r\n])

Reply:
The changes to SyntaxTree.fs are all of that nature.